arm-logic/2016-06-01/examples/SessionsListByIntegrationAccounts.json

```diff
@@ -0,0 +1,59 @@
+{
+  "title": "List by integration account session examples",
+  "parameters": {
+    "api-version": "2016-06-01",
+    "subscriptionId": "34adfa4f-cedf-4dc0-ba29-b6d1a69ab345",
+    "resourceGroupName": "testrg123",
+    "integrationAccountName": "testia123"
+  },
+  "responses": {
+    "200": {
+      "headers": {
+        "Cache-Control": "no-cache",
+        "Pragma": "no-cache",
+        "Content-Length": 1328,
+        "Content-Type": "application/json; charset=utf-8",
+        "Expires": -1,
+        "Vary": "Accept-Encoding",
+        "Date": "Tue, 21 Feb 2017 20:43:03 GMT"
+      },
+      "body": {
+        "value": [
+          {
+            "properties": {
+              "createdTime":"2017-02-21T23:06:24.0152955Z",
+              "changedTime":"2017-02-21T23:06:24.0158575Z",
+              "content":"123"
+            },
+            "id":"/subscriptions/34adfa4f-cedf-4dc0-ba29-b6d1a69ab345/resourceGroups/testrg123/providers/Microsoft.Logic/integrationAccounts/testia123/sessions/IntegrationAccountSession1662",
+            "name":"IntegrationAccountSession1662",
+            "type":"Microsoft.Logic/integrationAccounts/sessions",
+            "etag": "\"08587138885014622716\""
+          },
+          {
+            "properties": {
+              "createdTime": "2017-02-21T23:06:23.7535558Z",
+              "changedTime": "2017-02-21T23:06:23.7540113Z",
+              "content": "123"
+            },
+            "id": "/subscriptions/34adfa4f-cedf-4dc0-ba29-b6d1a69ab345/resourceGroups/testrg123/providers/Microsoft.Logic/integrationAccounts/testia123/sessions/IntegrationAccountSession6808",
+            "name":"IntegrationAccountSession6808",
+            "type":"Microsoft.Logic/integrationAccounts/sessions",
+            "etag":"\"08587138885017240103\""
+          },
+          {
+            "properties": {
+              "createdTime":"2017-02-21T23:06:23.8367926Z",
+              "changedTime":"2017-02-21T23:06:23.8370161Z",
+              "content":"123"
+            },
+            "id": "/subscriptions/34adfa4f-cedf-4dc0-ba29-b6d1a69ab345/resourceGroups/testrg123/providers/Microsoft.Logic/integrationAccounts/testia123/sessions/IntegrationAccountSession7315",
+            "name": "IntegrationAccountSession7315",
+            "type": "Microsoft.Logic/integrationAccounts/sessions",
+            "etag": "\"08587138885016407765\""
```

Comment:
"etag" not defined in swagger https://travis-ci.org/Azure/azure-rest-api-specs/jobs/204011630#L2340